Update strings.xml

kendraralston opened the pull request:
Edited copy.

kendraralston committed:
Update strings.xml

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -4,15 +4,15 @@
     <!-- Login activity -->
     <string name="app_title">beeline</string>
-    <string name="tag_line">Direct path to highest mutual attraction.</string>
+    <string name="tag_line">Your direct path to highest mutual attraction.</string>
```
That's the branding tagline. It cannot be edited.

The tagline goes with all the marketing.

There is an argument that calling out the user with "your" is a nice touch. Though because it is the tagline, e.g. meant for a business card and a poster at a convention, it's already pretty wordy. Adding "your" makes it to my eyes so wordy that I'm less likely to actually read it when I'd path a convention booth.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -4,15 +4,15 @@
     <string name="rate_users_title">Pick the one you like better</string>
     <string name="rate_users_body">
-        Let the algorithm learn the nuanced landscape of your dating preferences versus the general yes/no buckets of swiping.
+        Let our algorithm determine the nuanced landscape of your dating preferences.
```
Simplification is a good thing.

The wording is very old.

Remember, the comment that I made on a Google doc: A Beeline differentiator is that there is no algorithm that decides whom the user matches with. It's entirely decided by the users. The matchmaker called it with excitement: "You are the algorithm." So, the copy shouldn't say "our algorithm."

(Technically, every program is an algorithm. In common language, algorithms are meant to be complex logic that make allegedly intelligent decisions. The app doesn't make matching decisions of its own in that sense.)

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -21,15 +21,15 @@
     <string name="get_matched_body">
         Tired of hundreds of matches that go nowhere? Based on your ranking, we know exactly who
-        is totally hot for who and you feel the same way.
+        is into you.
```
Good simplification.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -21,15 +21,15 @@
-        Through mutual ranking, you will have much higher quality prospects. Further, everyone is
-        limited to contact only the 10 best matches. Start a genuine conversation with right people.
+        Through mutual ranking, you will have much higher quality prospects. Everyone is
+        limited to contact only the 10 best matches daily. Start a genuine conversation now!
```
Good simplifications.

The  "now!" is a good call to action.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -59,7 +59,7 @@
     <!-- No more pairings available activity -->
     <string name="no_more_pairings_heading">
-        No more pairings are available
+        No More Available Pairings
```
Did you go with title case instead of sentence case on purpose?

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -73,14 +73,14 @@
         match.\n
         \n
-        If you like the person, message them quickly. If you want someone "better," keep ranking
+        If you like the person, message them quickly. Not that into it? No worries. Keep ranking
```
Much more conversational and fun to read.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -91,11 +91,11 @@
     <string name="fuck_off_dialog_title">The famous fuck off button</string>
     <string name="fuck_off_dialog_text">
-        Warning! This action cannot be undone! You will never ever hear or see this person again.\n
+        Warning! This action cannot be undone! You will never hear from this person again.\n
```
I see what you are doing from a language perspective. An English teacher would correct "never ever under any circumstances even when hell freezes over" as redundant and replace it with "no." Sometimes, like when telling someone not to press the nuke launch button, a simple "no" doesn't quite do.

You can probably re-write the whole dialog message to deliver the message more effectively. So here is the goal for the message: Really discourage users from pressing the "fuck off button."

The reasons are this:
- A casual user may think that this works like on another dating app where the user gets more or less equivalent options. There discarding a match means that potentially something better pops up. Beeline doesn't work like that.

- In the regular course of actions, the comparisons of the user should already sort out whom the user is interested in. So the matches that they see really are matches that the users decided are their favorites.

- If the users are unhappy about these "favorites" because they didn't get liked back by their first choices, dismissing these choices will only get them worse choices. So that will set up a bad user experience. "Ugh, I hate Joe. Let me dismiss him. Oh no, Roger is even worse. Go away! Oh my god, what's wrong with Leo. Why do the choices keep getting worse and worse???" Well you already ranked your choices and your dismissing the ones on the top of your list!

- Also, the user should realize that they cannot undo this action ever. So tread carefully. We don't want people to pick around back and forth. We want users to give the top 10 matches their full attention and energy. (That's how we avoid the hundreds of messages going around with no or little response.)

- Ideally, this button only gets used after the users message each other and find out that the don't like each other. "Well, his profile was really exciting. But the BO on the date made even the flowers on the table wilt. I pass."


It's not necessary to explain all of this. The key objective is that the user is very hesitant to use the button, kind of understands the ranking impact, and is completely fine to dismiss users after they messaged/dated. Also, the encouragement should be to really focus the energy on the ten matches because they have the highest chance of going somewhere.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -117,36 +117,33 @@
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We\'ve all been in group situations, e.g. a class or summer camp. After a little while, the
```
You can re-write the about part from scratch. You don't have to limit yourself to simply editing the text. I wrote that with little planning and no structure.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -287,30 +283,29 @@
     <!-- Notifications -->
     <string name="updated_matches_notification_title">
-        Woohoo! Beeline has new matches for you!
+        Psst! Beeline has new matches for you!
```
What makes Pssst better than Woohoo?

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -287,30 +283,29 @@
     <string name="updated_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The algorithm just
-        finished running. You\'ve got some hot peeps to check out!
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The algorithm just
```
I think both versions of the text are too long to actually show up in the system dialog for notifications. I'll try to get a screenshot to show you later.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -287,30 +283,29 @@
-        the first message, the match becomes locked. (3) This is the hardest yet most satisfying.
-        Become more attractive. Hit the gym. Get new clothes. Put a smile on your face by following
-        your passion.
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The last person
```
This one turned out to be way too long either. I'll try to get you a screenshot of how little space in the Android system display is.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -287,30 +283,29 @@
     </string>
     <string name="new_message_notification_text">
-        Check out what a cutie is writing you!
+        Check out what someone's writing you!
```
Isn't a sweet name a lot more exciting than the generic someone? "Hottie alert" gets me a lot more giddy than "Alert." (Not saying to use the word hottie specifically.)

thomasfischersm reviewed (changes requested):
See comments inline.

kendraralston committed:
Update strings.xml

Minor edits -- haven't addressed all of the comments yet.

kendraralston committed:
Update strings.xml

These are the newest changes. I believe I addressed all of your comments, but I'm not too familiar with GitHub so apologies if I missed something! Please let me know.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -127,19 +127,14 @@
-        group branches off into pairs. Jenny, the runner, goes
-        with Fred, also a runner. James always watches indie flicks. He\'ll end up with April
-        because she does the same.\n
+        We've all been there: messages that go nowhere; likes, winks, and pokes sent with no response; 
```
These two lines are really good content wise.

thomasfischersm replied:
The content of frustration of interaction with low quality matches is a good start to segue into high mutual attraction. You expressed it with good conversation tone ("We've all been there") and added sentences phrases that capture the online experience (e.g. "messages that go nowhere").

Nitpicking, the content of the list could be improved. Having a semi-colon list with comma sub lists is good for an article or longer writing. For the screen writing, it seems a bit too complicated to catch the attention. Also, the particular list items could be stronger. Nowadays, many people know that pokes and winks usually go nowhere. Likes are often only visible to premium subscribers of the online app. That makes it unsurprising to the avid online dater. Yet, in the area of online dating, you have a lot of other choices that actually happen and resonate with people.

Creepy messages are annoying. Getting a million "Hi" and "how are you messages is annoying. Messaging forever without being asked out is annoying. Spending hours after hours swiping and getting nowhere is a pain.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -127,19 +127,14 @@
-        Some of us won\'t settle for anything less than chasing unattainable "gods" and
-        "goddesses." Beeline slots you with only a small number of matches that you actually have
-        a chance with.\n
+        Fortunately, Beeline's got your back. It slots you with only a small number of matches that are 
```
"Fortunately, Beeline's got your back." is a great line to start expressing the actual difference.

The strength of Beeline is NOT that it matches with a small number of matches. There are other sites that match with a small number of matches. People are frustrated with them because the matches suck, and people aren't able to find other matches on their own in the app.

The difference is that Beeline knows through the comparisons exactly how much you are attracted to each person. Beeline also knows exactly who of your top choices is highly attracted back at you. Beeline then matches you only with the people that have the highest mutual attraction. So, these are the people that you have the best chance of starting something. (You and he/she expressed high attraction for the match. No fallible/weak/faulty algorithm made the decision.)

That was very verbose to clearly explain it. The copy should follow the short conversational tone that you started really well with "We've all been there... Fortunately, Beeline's got your back..."

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -127,19 +127,14 @@
-        Avoid wasting time by sending hundreds of messages with zero responses.
-        Avoid getting your inbox flooded by thousands of messages. Get focused right on a hand-
-        selected number of people that are your high value matches.
+        Avoid wasting time by sending hundreds of messages with zero responses. Get focused on a chosen few 
```
That's clear and short with a fun tone.

Nitpicks:
- "a chosen few" -> "your chosen few" No algorithm in the usual marketing sense made the decision. Both users decided on whom they like. It's good to restate the essence of "you are the algorithm."
- "create your own connections" seems a bit vague. Something that expresses a serious beginning of the dating process seems better.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -286,19 +281,14 @@
-        users to find the special one who\'s into you. (2) Text your matches faster. Once you send
-        the first message, the match becomes locked. (3)Become more attractive. Put a smile on your face
-        by following your passion.
+        Don't worry! Rank more profiles to find who likes you back.
```
Good. Short and sweet.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -91,15 +91,15 @@
-        going to give you the next person down your list. If you ranked them right, you are going to
+        Please, think long and hard before proceeding. Based on your preferences, we\'ve given you
+        the best match that\'s interested in you. If you reject this person, you'll get
+        the next person on your list. If you ranked them right, you are going to
```
"If you ranked them right, you are going to like them less." That's a great sentence to capture what the users should "get into their head."

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -91,15 +91,15 @@
         \n
-        If this isn\'t going anywhere, do the merciful thing. Throw this person back into the wild
-        and tell this person to fuck off!
+        But, if you're absolutely sure this isn\'t going anywhere, then do the merciful thing. Throw this person back into the wild
```
I would have thought you'd edit that spur of the moment sentence completely out. As drastic as it is, it is kind of very fun. My girlfriend laughed at it very hard as well.

You touched it up nicely!

kendraralston replied:
I debated removing it, but it really adds a nice touch!

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -117,36 +117,28 @@
-        chances are that your top choices will run out of matches before you\'d even get considered.
-        You\'ll likely end up matching with other average attractive people who couldn\'t get their
-        first choices.\n
+        matched up unless either of you runs out of matches.\n
```
I like the direction of shortening the text. Yet the text seems to jump around topics too fast without explaining the user enough to understand what's going on.

What does it mean to run out of matches? That probably needs a bit more context.

The way how the "niche" is introduced wouldn't make sense to me. You kind of have the words and phrases there, yet to the uninitiated, it doesn't seem to introduce it in a comprehensible way.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -183,8 +175,7 @@
         \n
-        Remember, only an about Twitter sizes message will show up during profile comparison. Of
-        course, you can add more text for actual matches.
+       Of course, you can add more text for actual matches.
```
This was edit it so much that the actual message was lost.

The paragraph had two points:
1. The profile (shown to potential matches) is short. The user has to diligently choose what the user writes in it.
2. The profile is also shown to actual matches. (The actual matches see a full screen profile view instead of the half screen comparison version.) In that full screen version, the full length of the profile is shown.

In effect, the user can choose to write the brief 1.5 line profile. That will show up on the profile comparison screen to potential matches. Additionally, the user could add more profile text. That won't show up during comparison, but will be visible later. It's a bit of a way to have a pseudo private section of the profile that's only visible to actual matches.

In other words, the main contents is:
1. Choose your words carefully only a short Twitter-size message is visible during profile comparison.
2. Feel free to add additional text. Your actual matches will be able to see that.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -256,7 +247,7 @@
     <!-- Send Us Feedback option -->
     <string name="feedback_dialog_title">Send us feedback!</string>
-    <string name="feedback_dialog_message">An e-mail will be opened where you can send me feedback.</string>
+    <string name="feedback_dialog_message">An e-mail will open where you can send me feedback.</string>
```
Good.

thomasfischersm reviewed (changes requested):
Have you looked at the text under the line "<!-- Feature discovery -->"? I thought you'd make edits to it. I worked pretty hard on making the text small enough to fit the screen yet failed to find a good solution.

kendraralston commented:
"Have you looked at the text under the line ""? I thought you'd make edits to it. I worked pretty hard on making the text small enough to fit the screen yet failed to find a good solution." --> can you point me to where this is? I'm not sure what you're referring to.

thomasfischersm commented:
Ah, GitHub removed text that looks like a comment. There is a comment in the file: "Feature discovery". Have you reviewed the text underneath it?

kendraralston committed:
Edits 9/10

I believe I addressed all of the comments in the PR. Feeling much better about providing solid information. I thought about the "feature discovery" and am wondering if I'll need to rework #4.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -117,24 +117,27 @@
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. You get matched based on whom you
+        The app limits every user to maximum 10 matches. You get matched based on whom you
```
"You get matched based on whom you matched up unless either of you runs out of matches."

That seems like circular logic: "matched based on whom you match up."

Also logically, the structure should probably be:
- Create stack ranking from comparisons.
- Match best mutual choices based on stack ranking.
- Stop matching after 10 matches.

(Right now it's: Rank, limit, then match.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -117,24 +117,27 @@
+        The app limits every user to maximum 10 matches. You get matched based on whom you
         like the most and who likes you back. If you both are super popular, chances are you\'ll get
-        matched up unless either of you runs out of matches.\n
+        matched up unless either of you runs out of matches. If you are more "average" attractive, chances 
```
The "average attractive" part is true. It's a bad message. We worked hard on that messaging with the FAQ to replace it with the idea of "local peaks" or more conversational people who are attracted to each other's niche. E.g., the bit about punks that follows.

kendraralston replied:
This one is tricky. I've reworded it a bit, but I'll need to come back to it and flesh it out.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -117,24 +117,27 @@
-        genuinely interested in you.\n
+        Fortunately, Beeline's got your back. Through your comparisons, it knows exactly how much you are attracted to each person 
+        and who is highly attracted back. Beeline then matches you ONLY with 
+        people who have the highest mutual attraction. Think mutual control, likes, and potential. \n 
```
This is all very strong and excels at condensing a lot with a conversational and fun tone.

"Think mutual control, likes, and potential."
This sentence is misleading. The app should be distanced from "likes." People hand those out casually. The app also doesn't calculate a potential OKCupid and eHarmony advertise algorithms based on statistics and psychology research to predict potential. Beeline doesn't put stock in those algorithm predictions but trusts the gut instinct/reaction of the users.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -175,7 +178,8 @@
         dogs\n
         \n
-       Of course, you can add more text for actual matches.
+       Remember: Choose your words carefully. Potential matches wil see only a short Twitter-size message.
```
Reads clear and well.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -265,10 +269,10 @@
     <string name="app_info_body">
-        1. Choose between two users repeatedly.\n
-        2. The app creates a stack ranking.\n
+        1. You choose between two users repeatedly.\n
```
Character count is important with this one. The brief has a screenshot. Space is very limited. Plus, users won't take the time to read it. They'll scan it.

Ideally, you can decrease the word count/letter count.

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -265,10 +269,10 @@
+        2. The app then creates a stack ranking.\n
         3. A daily algorithm matches for mutual attraction.\n
-        4. Everyone is limited to the top 10 matches.
+        4. Enjoy your top 10 matches!
```
It's great that it's short, but it also seems to lose meaning. The new version leaves open how many total matches the user gets. It could be that the user gets 100 matches and the text suggests to enjoy the first 10. The message of limiting is edited out.

E.g. "Here is your new car for your 18th birthday. Enjoy the first 1,000 miles. They'll make a great memory." vs "Your warranty runs out after 1,000 miles.

kendraralston replied:
Tried to limit character count here and reword #4.

thomasfischersm reviewed (changes requested):
Pretty good. Some more comments.

kendraralston committed:
Another round of edits

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -269,10 +264,10 @@
-        3. A daily algorithm matches for mutual attraction.\n
-        4. Enjoy your top 10 matches!
+        1. Choose between two profiles repeatedly.\n
+        2. App creates profile rankings.\n
```
I wonder if there is a way to tie #2 to #1, i.e. to suggest that the profile comparisons result in the ranking rather than the app deciding on the ranking.

"Stack ranking of profiles is inferred." (clumsy wording)

thomasfischersm commented on app/src/main/res/values/strings.xml:
```diff
@@ -269,10 +264,10 @@
-        4. Enjoy your top 10 matches!
+        1. Choose between two profiles repeatedly.\n
+        2. App creates profile rankings.\n
+        3. Algorithm matches for mutual attraction.\n
```
I wonder if there is a way to edit "algorithm" out for the no well-known reasons.

"Daily update of matches with highest mutual attraction." (way too long to fit in to the UI)
"Matches daily updated by mutual attraction."

Maybe, there is something to be done by changing the fourth line as well.

"3. Matches updated daily."
"4. Matches limited to the top 10 mutual attraction matches" (way too many words)

"3. Matched by highest mutual attraction daily."
"4. Limited to the 10 highest quality matches."


Hm, maybe there is an entirely different spin by changing all bullet items:
1. Choose between two profiles repeatedly.
2. Repeated winners become your top choices.
3. Win his/her comparisons as well.
4. Only the top 10 most mutual connections get matched.
(This version doesn't sound very good. Though, you may see how by changing all four together, there may be a really strong version hiding.)

thomasfischersm reviewed (changes requested):
See comments.

kendraralston committed:
Update strings.xml

I liked your second option -- more of a call to action. Here's what I've come up with so far.
